fe/fe-core/src/main/java/org/apache/doris/planner/DistributedPlanner.java

```diff
@@ -812,6 +843,7 @@ private PlanFragment createSetOperationNodeFragment(
         }
 
         // There is at least one partitioned child fragment.
+        // TODO(ML): here
```

Comment:
TODO what？

Reply:
Used to mark the location of subsequent colocate plan work. But it has now been developed. no need